apps/cowswap-frontend/src/common/updaters/orders/ExpiredOrdersUpdater.ts

```diff
@@ -35,14 +33,9 @@ export function ExpiredOrdersUpdater(): null {
 
         // Filter orders:
         // - Owned by the current connected account
-        // - Created in the last 5 min, no further
         // - Not yet refunded
-        const pending = expiredRef.current.filter(({ owner, creationTime: creationTimeString, refundHash }) => {
-          const creationTime = new Date(creationTimeString).getTime()
-
-          return (
-            owner.toLowerCase() === lowerCaseAccount && now - creationTime < EXPIRED_ORDERS_PENDING_TIME && !refundHash
-          )
+        const pending = expiredRef.current.filter(({ owner, refundHash }) => {
+          return owner.toLowerCase() === lowerCaseAccount && !refundHash
```

Comment:
Actually, this might be a problem, because we'll forever try to update expired orders that are not ETH FLOW (don't have `refundHash`), and that's why this check was there in the first place.

Maybe it should only NOT have the limit when it's an ETH FLOW order.

Reply:
Nice catch!
Added a check: https://github.com/cowprotocol/cowswap/pull/3222/commits/1a06dd71394520339c2a914845b9d0b3d04383d5

Reply:
But what if an order is expired, and we don't know if its traded or not trade? 
we created the order, closed the browswe and came back 1h later

will we update the order and get the right state?

Reply:
Yes, we will.
It's actually this case: https://github.com/cowprotocol/cowswap/pull/3222#issuecomment-1761294009